akka-docs/src/test/java/jdocs/persistence/PersistenceMultiDocTest.java

```diff
@@ -4,28 +4,84 @@
 
 package jdocs.persistence;
 
+import akka.persistence.AbstractPersistentActor;
 import akka.persistence.UntypedPersistentActor;
+import com.typesafe.config.Config;
+import com.typesafe.config.ConfigFactory;
 
 public class PersistenceMultiDocTest {
 
     //#default-plugins
-    abstract class ActorWithDefaultPlugins extends UntypedPersistentActor {
+    abstract class ActorWithDefaultPlugins extends AbstractPersistentActor {
         @Override
-        public String persistenceId() { return "123"; }
+        public String persistenceId() {
+            return "123";
+        }
     }
     //#default-plugins
 
     //#override-plugins
-    abstract class ActorWithOverridePlugins extends UntypedPersistentActor {
+    abstract class ActorWithOverridePlugins extends AbstractPersistentActor {
         @Override
-        public String persistenceId() { return "123"; }
+        public String persistenceId() {
+            return "123";
+        }
+
         // Absolute path to the journal plugin configuration entry in the `reference.conf`
         @Override
-        public String journalPluginId() { return "akka.persistence.chronicle.journal"; }
+        public String journalPluginId() {
+            return "akka.persistence.chronicle.journal";
+        }
+
         // Absolute path to the snapshot store plugin configuration entry in the `reference.conf`
         @Override
-        public String snapshotPluginId() { return "akka.persistence.chronicle.snapshot-store"; }
+        public String snapshotPluginId() {
+            return "akka.persistence.chronicle.snapshot-store";
+        }
     }
     //#override-plugins
 
+    //#runtime-config
+    abstract class ActorWithRuntimeConfig extends AbstractPersistentActor {
```

Comment:
to follow the naming conventions of others (e.g. `AbstractPersistentActorWithTimers`) this should be `AbstractPersistentActorWithRuntimeConfig`.

Reply:
Thanks, we will rename this to match the current naming convention